lib/components/sidebar-component.js

```diff
@@ -41,6 +41,9 @@ export default class SidebarComponent extends BaseContainer {
 	selectAllSites() {
 		return driverHelper.clickWhenClickable( this.driver, By.css( '.all-sites a' ) );
 	}
+	selectViewThisSite() {
+		return driverHelper.clickWhenClickable( this.driver, By.css( '.site__content[title="View this site"]' ) );
```

Comment:
We try to avoid using English strings as locators, since a) these can and do change quite often, and b) this test couldn't run against non en locales, which we do for i18n testing on occasion. For example, in German the element looks like:

`<a class="site__content" href="http://watirmelon.blog" data-tip-target="site-card-preview" target="_blank" title="Diese Website ansehen" aria-label="Vorschau für watirmelon.blog-Website anzeigen">`

so something like:

`a[data-tip-target="site-card-preview"]`

should work